cli/run_server.py

```diff
@@ -32,16 +46,19 @@ def main():
     parser.add_argument('--min_batch_size', type=int, default=1,
                         help='Minimum required batch size for all expert operations')
     parser.add_argument('--max_batch_size', type=int, default=16384,
-                        help='The total number of examples in the same batch will not exceed this value')
+                        help='The total number of tokens in the same batch will not exceed this value')
+    parser.add_argument('--inference_max_length', type=int, default=None,
+                        help='Maximum total sequence length permitted per inference, defaults to max_batch_size tokens')
     parser.add_argument('--cache_dir', type=str, default=None, 
                         help='Path to a directory in which a downloaded pretrained model configuration should be cached if the standard cache should not be used.')
-    parser.add_argument('--cache_size_bytes', type=int, default=None,
-                        help='The size of memory cache for storing past attention keys/values between inference steps')
     parser.add_argument('--device', type=str, default=None, required=False,
                         help='all experts will use this device in torch notation; default: cuda if available else cpu')
     parser.add_argument("--torch_dtype", type=str, default="auto",
                         help="Use this dtype to store block weights and do computations. "
                              "By default, respect the dtypes in the pre-trained state dict.")
+    parser.add_argument('--attention_cache_bytes', type=str, default=None,
```

Comment:
```suggestion
    parser.add_argument('--attn_cache_size', type=str, default=None,
```

I strongly advise to replace `cache_bytes` to `cache_size` because:

- Specifying the size in bytes is standard across Python libs
- Moreover, here the size can be specified in any units

Also, I weakly advise to replace `attention` to `attn` because it's much shorter (but still understandable to everyone). This is optional though.

Reply:
[done both]